.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!-- Thank you for the pull request! -->
+
+#### Reference Issue
+<!-- Please provide a link to the respective issue on the issue tracker (https://github.com/cms-l1t-offline/cms-l1t-analysis/issues) if one exists. For example,
+
+Fixes #<ISSUE_NUMBER>
+-->
+
+
+#### What does this pull request implement/fix? Explain your changes.
+
+
+
+#### Any other comments?
```

Comment:
A common thing that might be worth including and therefore might be good to add as an example here, is when you need another pull request to be closed before this one.  Want to add something like this as a comment here?

Reply:
Thanks! Added example.